index.html

```diff
@@ -0,0 +1,484 @@
+            <div class="first-level-hover-wrapper">
+              <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                  <img src="/img/management.png" alt="Claim Management Web Design Example">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Management</strong></p>
+              <p>UI design, brand & logotypes</p>
+            </figcaption>
+          </figure>
+          <figure class="l-flex-item f-small-line-height">
+            <div class="first-level-hover-wrapper">
+              <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                   <img src="/img/korean.png" alt="Korean Restaurant Web Design Example">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Korean beef</strong></p>
+              <p>Webdesign, brand & logotypes</p>
+            </figcaption>
+          </figure>
+          <figure class="l-flex-item f-small-line-height">
+            <div class="first-level-hover-wrapper">
+              <div class="item-wrapper">
+                <div class="s-portfolio-image">
+                   <img src="/img/facebook_thumbs.png" alt="Facebook Icon Design Example">
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Facebook thumbs</strong></p>
+              <p>Icons design, brand & logotypes</p>
+            </figcaption>
+          </figure>
+        </div>
+      </article>
+      <article class="b-black" id="about">
+        <h2 class="f-bold f-large">ABOUT</h2>
+        <section class="l-flex-container">
+          <p class="l-flex-item l-about-txt">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+          <p class="l-flex-item l-about-txt">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">SKILLS</h3>
+          <p class="s-progress-paragraph">PHOTOSHOP</p><progress min="0" max="100" value="80"></progress></p>
```

Comment:
You also have an extra closing `</p>` tag on this (and all of the following `progress` lines). Generally the browser will ignore this, but it could cause headaches later on, and it's also confusing to anyone else who might step in to work on the project -- should the paragraph end after the text? Or should it include the `progress` element?